cmd/launcher/launcher.go

```diff
@@ -205,6 +206,25 @@ func runLauncher(ctx context.Context, cancel func(), opts *launcher.Options) err
 	controlService.RegisterConsumer("kolide_desktop_menu", runner)
 	controlService.RegisterSubscriber("kolide_desktop_flags", runner)
 
+	// Run the notification service
+	notificationConsumer, err := notificationconsumer.NewNotifyConsumer(
+		db,
+		runner,
+		ctx,
+		notificationconsumer.WithLogger(logger),
+	)
+	if err != nil {
+		return fmt.Errorf("failed to set up notifier: %w", err)
+	}
+	// Runs the cleanup routine for old notification records
+	runGroup.Add(notificationConsumer.Execute, notificationConsumer.Interrupt)
+
+	if controlService != nil {
```

Comment:
Can remove the if block now that `controlService` is always expected to be non-nil

Reply:
Oh yup, thank you!